### DISCUSSION TITLE
how can i use html label in my file?

### DISCUSSION THREAD

**Discussion by hbamboo** [marp-team/marp#189] [Q&A] [answered]
For example,if I want to use ![image](https://user-images.githubusercontent.com/49580879/138117750-2be507d2-b519-433a-9c00-2ef5c090fc6a.png) ,how can i make it work?
It just showed as a string like '
![image](https://user-images.githubusercontent.com/49580879/138117902-cb2ec207-2a39-4a3a-96f8-55fb5ac1f5de.png)
'.

**Comment by yhatt** [comment, marked as answer]
Did you read the extension README?
https://github.com/marp-team/marp-vscode#enable-html-in-marp-markdown-%EF%B8%8F

**Reply by hbamboo** [reply]
Thanks for the reply, I tried to find it in the wrong place.

**Comment by hxy1027** [comment]
Have you solved it yet
